addons/medical/stringtable.xml

```diff
@@ -44,12 +45,15 @@
       </Key>
       <Key ID="STR_ACE_Medical_Transfuse_Plasma">
         <English>Transfuse Plasma</English>
+        <Polish>Przetocz osocze</Polish>
       </Key>
       <Key ID="STR_ACE_Medical_Transfuse_Saline">
         <English>Transfuse Saline</English>
+        <Polish>Przetocz solankę</Polish>
```

Comment:
Solanka kojarzy mi się po prostu z wodą z dodatkiem soli :) Nawet takie wyniki otrzymałem w Google po wpisaniu solanka (solanka to też zupa). Bardziej pasowałoby chyba płyn fizjologiczny.